Makefile

```diff
@@ -194,7 +194,7 @@ pylint: $(PYSOURCES) $(wildcard tests/*.py)
 		setup.py khmer/[!_]*.py khmer/__init__.py scripts/*.py tests \
 		oxli/*.py || true
 
-pylint_report.txt: ${PYSOURCES} $(wildcard tests/*.py)
+pylint_report.txt: ${PYSOURCES} $(wildcard tests/*.py) $(wildcard sandbox/*.py)
```

Comment:
Should this be part of this PR? I don't see any harm in it, just wanted to check.